.eslintrc

```diff
@@ -0,0 +1,23 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "standard",
+    "standard-react"
```

Comment:
I think we should provide only eslint's infrastructure rather than rules in this template as the rules should be determined by each team using this template. So let's remove the `extends` part.